### PR TITLE
Add a common column wrapper to be used with lists

### PR DESCRIPTION
**Pull request by ghost**
Currently, Orckestra apps use a stand alone ColumnWrapper.  This common one can be provided to allow changes to be done for all applications.

### CODE REVIEW

File: docs/components.md [hunk 1]
```diff
@@ -43,6 +43,28 @@ See also the more [detailed documentation for list components](lists.md).
 
 Shows a pretty checkbox. The same props are accepted as for `<input type="checked" />` elements. Use `value` for whether the checkbox is checked or not, rather than `checked`. If no `id` is passed, one will be generated and used.
 
+## ColumnWrapper
+
+This is a default column wrapper available to be used by the applications.  This will ensure all applications experience the same behavior for columns used in lists.
```
Comment (gertsonderby) [inline docs/components.md:48]: This is not for list columns, but for column layouts. Best to remove that ambiguity.

File: docs/components.md [hunk 2]
```diff
@@ -43,6 +43,28 @@ See also the more [detailed documentation for list components](lists.md).
...
+Here is some sample code on usage:
+
+> ```
+> const LocationList = () => {
+> 	const scope = useSelector(getCurrentScope);
+> 	const name = "locationList" + scope;
+> 	const locations = unwrapImmutable(useSelector(mappedLocationListSelector));
+> 	useLoader(getLocations(scope), locationListLength);
+> 
+> 	return (
```
Comment (gertsonderby) [inline docs/components.md:59]: The stuff above this line is probably not relevant for the example.

Comment (gertsonderby) [reply docs/components.md:59]: Likewise the function closing, line 64 and 65, plus then you can remove a lot of whitespace.

**Comment by TONYFECTEAU** [comment]
This pull request will be closed.  I have a new branch and pull request with the proper user name.

**Pull request closed without merging**